python/ray/util/client/worker.py

```diff
@@ -94,16 +94,11 @@ def __init__(self,
                 # RayletDriverStub, allowing for unary requests.
                 self.server = ray_client_pb2_grpc.RayletDriverStub(
                     self.channel)
-                # Now the HTTP2 channel is ready, or proxied, but the
-                # servicer may not be ready. Call is_initialized() and if
-                # it throws, the servicer is not ready. On success, the
-                # `ray_ready` result is checked.
-                ray_ready = self.is_initialized()
-                if ray_ready:
-                    # Ray is ready! Break out of the retry loop
-                    break
-                # Ray is not ready yet, wait a timeout
-                time.sleep(timeout)
+                # Initialize the streams to finish protocol negotiation.
+                self.data_client = DataClient(self.channel, self._client_id,
```

Comment:
The constructor of dataclient doesn't look like it actually does anything, so this won't suffice as a health check. How about we just add a dummy ping RPC instead and preserve the original code structure here.

Reply:
Dumb question - what's the easiest way to implement a ping here? Should I modify the protobuf struct to have a dummy value?